src/index.css

```diff
@@ -419,13 +423,13 @@ user-select: none;
 }
 
 .next-sector-popup .mapboxgl-popup-content {
-    background: transparent;
+    background: rgba(90, 90, 90, 0.5);
     border-radius: 1.5px;
     /* color: white; */
     padding: 0;
     margin: 0;
     height: 3em;
-    width: 16em;
+    width: 250px !important;
```

Comment:
`!important` should be considered as a last resort system, but I think it's fine for a prototype.